Algoritmer/Grafer/heap_sort.md

```diff
@@ -40,6 +46,10 @@ Algoritmen utnytter altså at max-heapify alltid finner maximum i heapen og bruk
 
 ## Kjøretid og utregning
 <!-- Under ulike omstendigheter -->
+
+The HEAPSORT procedure takes time $O(n \log n)$ since the call to BUILD-MAXHEAP takes time $O(n)$and each of the $n-1$ calls to MAX-HEAPIFY takes
+time $O(\log n)$
```

Comment:
Missing a space, which can create render issues. Also guessing since this is in English its a quote? Also removed newline between L50 and L51, merging them.

```suggestion
> The HEAPSORT procedure takes time $O(n \log n)$ since the call to BUILD-MAXHEAP takes time $O(n)$ and each of the $n-1$ calls to MAX-HEAPIFY takes time $O(\log n)$
```